lib/Backends/Interpreter/tests/InterpreterOperatorTest.cpp

```diff
@@ -31,4 +31,5 @@ std::set<std::string> glow::backendTestBlacklist = {
     "EmbeddingBagByteRowwiseOffsets_Float_End_Offset_Partial/0",
     "EmbeddingBagByteRowwiseOffsets_Float16_AccumFloat_End_Offset_Partial/0",
     "EmbeddingBagByteRowwiseOffsets_Float16_AccumFloat16_End_Offset_Partial/0",
+    "BBoxTransform_Float16/0",
```

Comment:
Why is this necessary? The interpreter should support this, no?

Reply:
When I tested it failed - I think it is related to the next change you commented on:
getParent()->uniqueType(ElemKind::FloatTy, {imInfoDims[0]});